code/modules/mob/living/carbon/alien/say.dm

```diff
@@ -15,11 +15,21 @@
 	if(copytext(message,1,2) == "*")
 		return emote(copytext(message,2))
 
+	var/needtohack = 1 //hacky way to get around stupid byond shit
 	var/datum/language/speaking = parse_language(message)
+
+	if(speaking == null)
+		var/quickcheck = get_default_language()
+		if(quickcheck == null)
+			speaking = all_languages["Xenomorph"] //this is the hackiest thing, no clue how to avoid it
+			needtohack = 0
+		else
+			speaking = get_default_language()
+			needtohack = 0
+
 	if(speaking)

```

Comment:
There are three values in BYOND that are false: `0`, `""`, and `null`.

It should work properly; if you can show me a version that you think should work but doesn't, I'll take a look and see if I can spot anything.